### PR TITLE
hw5.3

### PR DESCRIPTION
**Pull request by OlessiaKharchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +15,28 @@
...
+
+        try {
+            List<String> listOfLines = Files.readAllLines(Paths.get(fileName));
+            List<String> resultList = new ArrayList<>();
+
+            for (String line : listOfLines) {
+                String[] arrayOfWords = line.split(" ");
+                for (String word : arrayOfWords) {
+                    if ((word.substring(0, 1)).equalsIgnoreCase(LETTER)) {
+                        resultList.add(word.replaceAll("[^A-Za-zА-Яа-я0-9]", "").toLowerCase());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:30]:
```suggestion
                        resultList.add(word.replaceAll("\\W+", "").toLowerCase());
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +15,28 @@
...
+                    }
+                }
+            }
+            Collections.sort(resultList);
+            return resultList.toArray(new String[resultList.size()]);
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:37]: let's rethrow this as a RuntimeException

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -8,7 +15,28 @@
...
+        }
+
+        return new String[]{};
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:40]: after fixing the previous comment, you will be able to remove this line

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (OlessiaKharchenko) [reply src/main/java/core/basesyntax/FileWork.java]: done!

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (OlessiaKharchenko) [reply src/main/java/core/basesyntax/FileWork.java]: done!

File: src/main/java/core/basesyntax/FileWork.java (hunk 3 above)
Comment (OlessiaKharchenko) [reply src/main/java/core/basesyntax/FileWork.java]: done!

**Pull request closed without merging**